src/main/java/io/ark/core/responses/AccountResponse.java

```diff
@@ -0,0 +1,21 @@
+package io.ark.core.responses;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.ark.core.model.Account;
+import io.ark.core.model.Delegate;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AccountResponse {
+  
+  private boolean success;
+  private Account account;
```

Comment:
@dafty This work is actually not in conflict with what you have. We can replace these `Account` and `Delegate` etc. in these files with the response objects you have created and use the model classes functionally. We might just have to add a way to convert from Response -> Model through their constructor or something?

Reply:
Can do - if we have lots of utility functions (so far, I have only added `getLocalDate()` in `Block` and `Transaction`) then it may make sense to have separate models and keep the responses clean, otherwise we can consider sticking with the response objects. Another argument is if we were introduce persistence, then separate models is the way to go.